frontend/appflowy_flutter/lib/plugins/document/presentation/editor_configuration.dart

```diff
@@ -163,6 +164,16 @@ Map<String, BlockComponentBuilder> getEditorBuilderMap({
       configuration: configuration,
     ),
     OutlineBlockKeys.type: OutlineBlockComponentBuilder(
+      configuration: configuration.copyWith(
+        placeholderTextStyle: (_) =>
+            styleCustomizer.workspaceOverviewBlockPlaceholderStyleBuilder(),
```

Comment:
Did you add this wrongly to the outline block instead of the overview block?

Reply:
Yeah, it seems I added it mistakenly to the outline block instead of the overview block